fe/fe-core/src/main/java/com/starrocks/analysis/FunctionParams.java

```diff
@@ -116,6 +163,13 @@ public void readFields(DataInput in) throws IOException {
                 exprs.add(Expr.readIn(in));
             }
         }
+        if (in.readBoolean()) {
```

Comment:
in which case will the problem happen? I don't know when to use these two functions.

Reply:
there are 4 ways to process it:
1. it seems no need to write/read() for `orderByElements`, as array_agg(x order by y) does not appear in catalogs, like aggregated table. otherwise,
2. as the current code, reasons are in comments.
3. change `orderByElements` to be derived from expr class
4. change `orderByElements` to be exprs in write(), and recovery to `orderByElements` in read().

Reply:
ok